app/controllers/to_change_everything_controller.rb

```diff
@@ -0,0 +1,33 @@
+class ToChangeEverythingController < ApplicationController
+  layout 'to_change_everything', only: [:show]
+
+  RTL_LANGUAGES = [
+    :ar, # Arabic
+    :ku, # Kurdish
+    :he, # Hebrew
+    :fa, # Persian
+  ].freeze
+
+  def show
+    http_cache_forever(public: true) do
```

Comment:
I am a little nervous about using `http_cache_forever`. It seems like the exact thing I want, but I don't think there is a way to bust the cache besides people manually hard-refreshing their browsers...

Reply:
err, maybe this isn't what I want. Seems like every new visitor will get 1-slow request.

I want 1 visitor to get 1 slow request, and then everyone to get the cached view afterwards

Reply:
Maybe we can use the usual caching system, but set a very long timeout for those pages.